app/code/Magento/CatalogImportExport/Model/Import/Product/Validator/Media.php

```diff
@@ -114,8 +114,8 @@ public function isValid($value)
                         ]
                     );
                     $valid = false;
+                    break;
```

Comment:
Maybe it would be better check all images and add messages for all invalid images
Now it just break after first invalid image
Please also change test accordingly.

Reply:
There is no sense to show all errors as they are not related to every image, but to row in csv in general.